index.js

```diff
@@ -1,30 +1,44 @@
 const score=document.querySelector(".score")
-let timeLeft=document.querySelector(".time-left")
+const timeLeft=document.querySelector(".time-left")
 const mole=document.querySelectorAll(".mole")
 const box=document.querySelectorAll(".box")
 
 let hitid=null;
+let totalScore=0;
 
-//remove the presence of class mole
+//initially remove the presence of any class mole 
 function clearMole(){
 mole.forEach((x)=>{
     x.classList.remove("mole")
 })
 }
-//make mole apppear randomly
+
+//make the mole apppear randomly
 function moleAppear(){
   let molelocation=box[Math.floor(Math.random() * 9)];
     molelocation.classList.add("mole")
+    //id of mole location is set to hitid variable
     hitid=molelocation.id
+    //mole is removed after 0.5s which will be the start of new mole appearence
+    setTimeout(() => {
+    molelocation.classList.remove("mole")
+    }, 500);
```

Comment:
`moleAppear` should not have to care about removing a mole. So It would be better to separate appearing from disappearing by creating two separate methods.